### ISSUE TITLE
removed .owl-controls 

### ISSUE THREAD

**Issue by volodymyrl** [smashingboxes/OwlCarousel2#892]
I've updated OwlCarousel2 to latest version and it's lost div.owl-controls for a navigation, and it broken my slider navigation. How can I add this wrap class by myself, and will you come it back?

**Comment by KennethAshley** [comment]
@volodymyrl can you please provide more information on your problem?

**Comment by eatarandom** [comment]
I'm experiencing the same issue I believe. The navigation components `owl-nav` and `owl-dots` are no longer wrapped with `owl-controls`.
```
<div class="owl-nav disabled">
  <div class="owl-prev">prev</div>
  <div class="owl-next">next</div>
</div>
<div class="owl-dots disabled">
  <div class="owl-dot active"><span></span></div>
  <div class="owl-dot"><span></span></div>
</div>
```
Demos show:
```
<div class="owl-controls">
  <div class="owl-nav disabled">
    <div class="owl-prev">prev</div>
    <div class="owl-next">next</div>
  </div>
  <div class="owl-dots disabled">
    <div class="owl-dot active"><span></span></div>
    <div class="owl-dot"><span></span></div>
  </div>
</div>
```

This is breaking the default css selections within `owl-theme`.

**Comment by hoppithek** [comment]
As far as I looked into it, it is the missing owl-theme class that is no longer automatically added to the sourrounding div of the complete carousel.

As a work-around.
changing `<div class="owl-carousel">...</div>` to `<div class="owl-carousel owl-theme">...</div>` will bring the controls back.

**Comment by greg5green** [comment]
This is correct -- `owl-theme` is a needed class as of https://github.com/smashingboxes/OwlCarousel2/commit/d33a9ee3b53e87dbf98f53def01342fc50189e33

The documentation was updated so the demos work, but this needs to be noted in the README and getting started areas of the documentation. Anyone up for a PR to change these?

**Comment by atilkan** [comment]
I would like to help but i break something. I just started exploring. Maybe later i can translate for turkish.

**Comment by greg5green** [comment]
:+1:

**Comment by spitza** [comment]
It'd be great if you made this more clear here (http://owlcarousel.owlgraphic.com/docs/started-installation.html). I initially didn't include the theme CSS in my head then couldn't seem to figure out why the nav controls weren't showing up. It would be helpful to say explicitly that you must include the theme CSS if you want controls, and include `"owl-theme"` as a class on the div.

**Comment by greg5green** [comment]
@spitza Feel free to submit a pull request.

**Comment by spitza** [comment]
Will do! Thanks.

**Comment by greg5green** [comment]
Documentation fixed by #975 and 030506e

**Comment by PureDude** [comment]
If someone needs the wrap necessarily...
https://stackoverflow.com/questions/44020865/wrapper-owl-controls-is-removed-in-owl-carousel-2
